docs/http/openfga.http

```diff
@@ -2,18 +2,44 @@
 @fga_store_id = 
 @fga_model_id = 
 
+## Required for Client Credentials Flows
+@fga_token_issuer = 
+@fga_api_audience = 
+@fga_client_id = 
+@fga_client_secret = 
+
+########################
+# GET API TOKEN FROM CLIENT CREDENTIALS FLOW
+########################
+
```

Comment:
Need to add `# @name auth` above line 15.

Otherwise, `@fga_api_token = {{auth.response.body.access_token}}` won't save properly.